### PR TITLE
Bump postgres versions used in CI (based on master repo)

### PR DESCRIPTION
**Pull request by pmwkaa**
Run tests on PG 12.6, 11.11, 10.16, 9.6.21

### CODE REVIEW

**Review by mkindahl — APPROVED** [review]

**Comment by erimatnor** [comment]
I believe it needs additional changes to pass pre-release tests (ignore `cluster` test on some PG versions).

**Review by erimatnor — APPROVED** [review]
Approving, assuming pre-release tests pass first before merge.

**Review by svenklemm — APPROVED** [review]

**Pull request merged**
